source/collection-management/timeseries-collection.json

```diff
@@ -0,0 +1,110 @@
+{
+  "description": "timeseries-collection",
+  "schemaVersion": "1.0",
+  "runOnRequirements": [
+    {
+      "minServerVersion": "4.9",
+      "serverParameters": {
+        "featureFlagTimeseriesCollection": {
+          "value": true,
+          "version": "5.0"
+        }
+      }
+    }
+  ],
+  "createEntities": [
+    {
+      "client": {
+        "id": "client0",
+        "observeEvents": [
+          "commandStartedEvent"
+        ]
+      }
+    },
+    {
+      "database": {
+        "id": "database0",
+        "client": "client0",
+        "databaseName": "ts-tests"
+      }
+    },
+    {
+      "collection": {
+        "id": "collection0",
+        "database": "database0",
+        "collectionName": "test"
+      }
+    }
+  ],
+  "initialData": [
+    {
+      "collectionName": "test",
+      "databaseName": "ts-tests",
+      "documents": []
+    }
+  ],
+  "tests": [
+    {
+      "description": "createCollection with all options",
```

Comment:
I've only added a single test to make sure drivers can create a collection with the `timeseries` option without error, and that the collection actually exists. 

I could add tests to expect errors from an empty `timeField` or an unsupported option (like `capped`) used in conjunction with `timeseries`, but the server will error in these cases. I could also add tests for inserting and finding on a `timeseries` collection, but from the drivers-side, these new collections function similarly to normal ones.

Reply:
Assuming the server strictly validates options for the `create` command and raises errors for unsupported options, I don't think we need an additional error test here.